emop-maven-plugin/src/main/java/edu/cornell/MonitorMojo.java

```diff
@@ -44,26 +44,26 @@ public void execute() throws MojoExecutionException {
         super.execute();
         getLog().info("[eMOP] Invoking the Monitor Mojo...");
         long start = System.currentTimeMillis();
-        generateNewMonitorFile();
+        Util.generateNewMonitorFile(getArtifactsDir() + File.separator + monitorFile, affectedSpecs);
         if (javamopAgent == null) {
             javamopAgent = getLocalRepository().getBasedir() + File.separator + "javamop-agent"
-                + File.separator + "javamop-agent"
-                + File.separator + "1.0"
-                + File.separator + "javamop-agent-1.0.jar";
+                    + File.separator + "javamop-agent"
+                    + File.separator + "1.0"
+                    + File.separator + "javamop-agent-1.0.jar";
         }
         Util.replaceFileInJar(javamopAgent, "/META-INF/aop-ajc.xml", getArtifactsDir() + File.separator + monitorFile);
         long end = System.currentTimeMillis();
         getLog().info("[eMOP Timer] Generating aop-ajc.xml and replace it takes " + (end - start) + " ms");
         if (!includeNonAffected || !includeLibraries) {
             start = System.currentTimeMillis();
             // Rewrite BaseAspect.aj to ignore non-affected classes
-            generateNewBaseAspect(includeNonAffected && includeLibraries);
+            generateNewBaseAspect();
```

Comment:
can you walk me through why the args were removed (I think I understand, but I want to be sure)

Reply:
The args aren't necessary because the methods called from `generateNewBaseAspect()` that handle the "exclude non affected classes" and "exclude libraries" part both check for the variables `includeNonAffected` and `includeLibraries` on their own.